### ISSUE TITLE
Integration with Github as issue tracker

### ISSUE THREAD

**Issue by adoptavia** [BadChoice/handesk#220]
Hi is it possible to integrate with Github instead of bitbucket?

**Comment by BadChoice** [comment]
Yes! you only need to create the IssuesManager and implement its methods, such as create issue,
I beliveve it shouldn't be hard, but right now I have no time for it

**Comment by AugmentBLU** [comment]
Is there anymore on this? Seems disjointed that the repo is hosted on Github yet there's no integration with Github. 

If there's no plans, can you let us know as we'll look at another solution.

**Comment by BadChoice** [comment]
We use it with bitbucket (it is where we have our private repos) so we are not planing on adding it, 
however feel free to submit a PR, it should be really easy as it is ready to use any Issue Tracker
